### ISSUE TITLE
Fails to Support Ember Octane?

### ISSUE THREAD

**Issue by EndangeredMassa** [ember-a11y/ember-a11y-testing#121]
Using ember-a11y-testing on an Ember Octane app, I see a lot of:

```
Error: Expect axe._selectorData to be set up
    at generateSelector (axe.js:4837)
    at Object.createUniqueSelector [as getSelector] (axe.js:4893)
    at DqElement.get selector [as selector] (axe.js:4265)
    at axe.js:3774
    at Array.map (<anonymous>)
    at axe.js:3767
    at Array.map (<anonymous>)
    at axe.js:3764
    at Array.forEach (<anonymous>)
    at Object.helpers.processAggregate (axe.js:3756)
```

It seems that this is a known issue of [axe-core #1041](https://github.com/dequelabs/axe-core/issues/1041). A workaround was implemented in https://github.com/ember-a11y/ember-a11y-testing/pull/110 but that doesn't appear to work in an Ember Octane app.

I don't know much about the details of the scheduler. @drewlee do you have any thoughts on how we could support Octane?

**Comment by drewlee** [comment]
@EndangeredMassa I haven't looked into Ember Octane yet, so I don't have direct insight into what the  problem may be. I'm assuming the error is coming up from the visual component audit feature? The scheduler is just a simple queue mechanism that ensures axe.run is executed synchronously. Otherwise, asynchronous execution will result in the exception that you printed out above. What would be the quickest way to reproduce the issue?

**Comment by rwjblue** [comment]
@EndangeredMassa - I _think_ that the fix from #110 should still work in octane apps, but the integration (at the moment) is based on `Ember.Component`/`@ember/component` which isn't used (too much) in an octane app (you'd use `@glimmer/component` instead).  In order to help track down the specific error you hit, would you mind pushing up a repo that demonstrates the error?

**Comment by drewlee** [comment]
@EndangeredMassa is this concern still valid? I've played around with a bunch of different versions of Ember w/Octane features and haven't been able to reproduce, at least not when setting up a new app. If this is still an issue, we'll need a reliable way to reproduce, otherwise I recommend closing the issue.
